Makefile

```diff
@@ -71,7 +71,7 @@ DOCKER_RUN_DOCKER := $(DOCKER_FLAGS) "$(DOCKER_IMAGE)"
 
 default: binary
 
-all: build ## validate all checks, build linux binaries, run all tests\ncross build non-linux binaries and generate archives
+all: build ## validate all checks, build linux binaries, run all tests cross build non-linux binaries and generate archives
```

Comment:
The `\n` are there for a reason. The `make help` action actually parses the comments, and formats them to print a nice help output;

```bash
make help

all                  validate all checks, build linux binaries, run all tests
                      cross build non-linux binaries and generate archives
binary               build the linux binaries
cross                cross build the binaries for darwin, freebsd and
                      windows
win                  cross build the binary for windows
tgz                  build the archives (.zip on windows and .tgz
                      otherwise) containing the binaries
deb                  build the deb packages
docs                 build the docs
gccgo                build the gcc-go linux binaries
install              install the linux binaries
rpm                  build the rpm packages
run                  run the docker daemon in a container
shell                start a shell inside the build env
test                 run the unit, integration and docker-py tests
test-docker-py       run the docker-py tests
test-integration-cli run the integration tests
test-unit            run the unit tests
validate             validate DCO, Seccomp profile generation, gofmt,
                      ./pkg/ isolation, golint, tests, tomls, go vet and vendor
manpages             Generate man pages from go source and markdown
help                 this help
```